website/docs/docs/dbt-cloud-apis/user-tokens.md

```diff
@@ -6,38 +6,44 @@ pagination_next: "docs/dbt-cloud-apis/service-tokens"
+The current user API key is located under **Personal Settings → API Key.** 
 
 Please [contact support](/docs/dbt-support#dbt-cloud-support) with any questions or concerns.
 
 :::
 
-## Account-scoped personal access tokens
+## Account-scoped Personal Access Tokens
 
 :::info New 
 
-On Feb 7, 2024, we introduced a new type of token for individual users called personal access tokens. Note that these differ from [Service Tokens or API Keys](/docs/dbt-cloud-apis/authentication#types-of-api-access-tokens). Before this release, user API keys were the only way to access dbt Cloud API on behalf of the user. These API Keys were user-specific and were not scoped to an account. To enhance the security of dbt Cloud, we are moving away from this model to account-specific tokens. 
+On Feb 7, 2024, we introduced a new type of token for individual users called Personal Access Tokens. Note that these differ from [Service Tokens or user API Keys](/docs/dbt-cloud-apis/authentication#types-of-api-access-tokens). 
+
+Before this release, User API keys were the only way to access dbt Cloud API on behalf of the user. 
+
+These API Keys were user-specific and were not scoped to an account. To enhance the security of dbt Cloud, we are moving away from this model to account-specific tokens. 
 
 :::
 
-Each dbt Cloud user with a [Developer license](https://docs.getdbt.com/docs/cloud/manage-access/seats-and-users) can create a new personal access token (PAT) to access the dbt Cloud API and dbt Cloud CLI. This token can execute queries against the dbt Cloud API on the user's behalf. To access dbt Cloud APIs and resources on behalf of the _account_, we recommend using service Tokens instead. Learn more about [which token type you should use](/docs/dbt-cloud-apis/authentication#which-token-type-should-you-use) to understand the token differences.
+Each dbt Cloud user with a [Developer license](https://docs.getdbt.com/docs/cloud/manage-access/seats-and-users) can create a new personal access token (PAT) to access the dbt Cloud API and dbt Cloud CLI. This token can execute queries against the dbt Cloud API on the user's behalf. To access dbt Cloud APIs and resources on behalf of the _account_, we recommend using Service Tokens instead. Learn more about [which token type you should use](/docs/dbt-cloud-apis/authentication#which-token-type-should-you-use) to understand the token differences.
 
 PATs inherit the permissions of the user that created them. For example, if a developer-licensed user with Project Admin role access to specific projects creates a PAT, the token will get the Project Admin role with access to the same projects as the user. These tokens are also account-specific, so if a user has access to more than one dbt Cloud account with the same email address, they need to create a unique PAT for each one of these accounts. 
 
-### Migrate from user API keys to personal access tokens
+### Migrate from user API keys to Personal Access Tokens
```

Comment:
@Millacy Heads up that we use [sentence case](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md#Titles) in title for a more conversational tone.
```suggestion
### Migrate from user API keys to personal access tokens
```